<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDipswitch</class>
 <widget class="QDialog" name="dlgDipswitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dipswitch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_Dipswitch"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Standard_Button">
     <item>
      <widget class="QPushButton" name="pushButton_Start">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/confirm.svgz</normaloff>:/icon/icons/confirm.svgz</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Default">
       <property name="text">
        <string>Default</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/overscan_default.svgz</normaloff>:/icon/icons/overscan_default.svgz</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
